tests/ui/specs/RangePicker/RangePicker-specs.js

```diff
@@ -0,0 +1,108 @@
+const Page = require('./RangePickerPage');
+const {extractValue} = require('./RangePicker-utils.js');
+
+describe('RangePicker', function () {
+
+	describe('LTR locale', function () {
+		beforeEach(function () {
+			Page.open();
+		});
+
+		describe('default', function () {
+			const rangePicker = Page.components.rangePickerDefault;
+
+			describe('5-way', function () {
+				it('should change the value forward when incrementing the rangePicker', function () {
+					browser.waitUntil(() => rangePicker.decrementer(rangePicker.self).isFocused(), {timeout: 1500,  interval: 100});
```

Comment:
Also, here please check these `wailtUntil` are mandatory or not.

Reply:
already on it 👍